### PR TITLE
[TASK] Remove outdated information

### PR DESCRIPTION
**Pull request by linawolf**
and orphaned pages that have been orphande for many years

### CODE REVIEW

File: Documentation/Home/References.rst [hunk 1]
```diff
@@ -88,28 +88,6 @@ Reference Manuals
                -  `12.4 <https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/>`__
                -  `11.5 <https://docs.typo3.org/m/typo3/reference-tsconfig/11.5/en-us/>`__
 
-   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
```
Comment (brotkrueml) [inline Documentation/Home/References.rst:91]: You are removing here the whole TypoScript card, not only the "TypoScript in 45 Minutes" link.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/Home/References.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Home/References.rst:91]: This whole card is about TypoScript in 45 Minutes as far as I can tell?

Comment (brotkrueml) [reply Documentation/Home/References.rst:91]: Nope, it is only linked in the text:
image
The "12.4" and "11.5" texts link to the TS reference.

Comment (linawolf) [reply Documentation/Home/References.rst:91]: woops, good catch!

File: Documentation/Home/References.rst [hunk 2]
```diff
@@ -88,28 +88,6 @@ Reference Manuals
                -  `12.4 <https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/>`__
                -  `11.5 <https://docs.typo3.org/m/typo3/reference-tsconfig/11.5/en-us/>`__
```
Comment (linawolf) [inline Documentation/Home/References.rst:90]:
```suggestion

   .. container:: col-md-6 pl-0 pr-3 py-3 m-0
   
      .. container:: card px-0 h-100

         .. rst-class:: card-header

            .. rubric:: :ref:`t3tsref:start`

         .. container:: card-body

            TypoScript is a configuration language that is specific to
            TYPO3. This is used to configure the frontend.

         .. container:: card-footer pb-0

            .. rst-class:: horizbuttons-striking-m

               -  `12.4 <https://docs.typo3.org/m/typo3/reference-typoscript/12.4/en-us/>`__
               -  `11.5 <https://docs.typo3.org/m/typo3/reference-typoscript/11.5/en-us/>`__
```

File: Documentation/Home/References.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/Home/References.rst:91]: reverted

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**